src/Symbol.spec.lua

```diff
@@ -11,7 +11,7 @@ return function()
 		it("should coerce to the given name", function()
 			local symbol = Symbol.named("foo")
 
-			expect(tostring(symbol):find("foo")).to.be.ok()
+			expect((tostring(symbol):find("foo"))).to.be.ok()
```

Comment:
Did StyLua do this?

Reply:
This is type analysis. `find` returns multiple values and `expect` only accepts one.